dds_web/commands.py

```diff
@@ -841,16 +841,25 @@ def monthly_usage():
         send_email_with_retry,
     )
 
+    # Get the instance name (DEVELOPMENT, PRODUCTION, etc.)
+    instance_name = flask.current_app.config.get("INSTANCE_NAME")
+
     # Email settings
     email_recipient: str = flask.current_app.config.get("MAIL_DDS")
     # -- Success
     email_subject: str = "[INVOICING CRONJOB]"
+    if instance_name:  # instance name can be none, so check if it is set and add it to the subject
+        email_subject += f" ({instance_name})"
+
     email_body: str = (
         "The calculation of the monthly usage succeeded; The byte hours "
         "for all active projects have been saved to the database."
     )
     # -- Failure
     error_subject: str = f"{email_subject} <ERROR> Error in monthly-usage cronjob"
+    if instance_name:  # instance name can be none, so check if it is set and add it to the subject
+        error_subject += f" ({instance_name})"
```

Comment:
We need to remove this part, the `instance_name` has already been added to the subject in line 851-853, so this will become `"[INVOICING CRONJOB] (DEVELOPMENT) <ERROR> Error in monthly-usage cronjob (DEVELOPMENT)"`
```suggestion
```